Parser.cpp

```diff
@@ -143,7 +143,7 @@ std::optional<std::string> commonItems::parser::getNextToken(std::istream& theSt
 			match->second(toReturn, theStream);
 			matched = true;
 		}
-		else if (toReturn.size() > 2 && toReturn[0] == '\"' && toReturn[toReturn.size() - 1] == '\"')
+		else if (toReturn.size() > 2 && toReturn.starts_with('\"') && toReturn.ends_with('\"'))
```

Comment:
Likewise, can remQoutes() be used here? I notice that this one is testing for >2 instead of >=2, so maybe there's a subtle issue. I can't think of one, though.

Also, it may be a better choice to just generate a stripped version of the lexeme right after getting it instead of stripping quotes up to three times.

Reply:
> Also, it may be a better choice to just generate a stripped version of the lexeme right after getting it instead of stripping quotes up to three times.

I think that would cause a problem if the registered keyword started and ended with a quote.

Reply:
> Likewise, can remQoutes() be used here?

It's already used here, but now I additionaly removed the condition and it still works.

Reply:
I described what I thought poorly, but @Zemurin is right that we should tread carefully. I won't advocate for changes that may have a performance impact at this time, not without devising a way of assessing the impact.